### PR TITLE
Changed location of Snakemake documentation in readthedocs

### PR DESCRIPTION
**Pull request by JessicaTiebel**
I moved the documentation of a automated parameter scan using snakemake to a more prominent position in read the docs.

- [x] Check if documentation compiles correctly.

### CODE REVIEW

**Comment by PrometheusPi** [comment]
@JessicaTiebel  please rebase (restart of CI is not enough)

File: docs/source/usage/parameter_scans.rst [hunk 1]
```diff
@@ -12,11 +12,13 @@ Each workflow consists of a ``Snakefile`` in which the workflow is defined using
 How to use
 """"""""""
 
-The example presented here performs several simulations with given input parameters.
+The example presented here performs several simulations with given input parameters with the LaserWakefield example of PIConGPU on hemera.
+
+#. Copy the ``Snakefile`` and ``config.yaml`` which are located in the PIConGPU source code here: ``picongpu/share/picongpu/examples/LaserWakefield/lib/python/snakemake/``. There also a ``requirements.txt`` and some exemplary parameters ``params.csv`` can be found which are compatible with the LaserWakefield example.
```
Comment (steindev) [inline docs/source/usage/parameter_scans.rst]: Missing comma and drop the 'also' please. You might want rephrase along the lines of 'In addition, a .... can be found there, which are ...'

File: docs/source/usage/parameter_scans.rst [hunk 2]
```diff
@@ -12,11 +12,13 @@ Each workflow consists of a ``Snakefile`` in which the workflow is defined using
...
 
 #. Make shure you have ``snakemake`` and the ``snakemake-executor-plugin-slurm`` installed and activated.
```
Comment (steindev) [inline docs/source/usage/parameter_scans.rst]: shure -> sure

File: docs/source/usage/parameter_scans.rst [hunk 3]
```diff
@@ -12,11 +12,13 @@ Each workflow consists of a ``Snakefile`` in which the workflow is defined using
...
 
-#. Adjust the profile ``config.yaml``:
+#. Adjust the profile ``config.yaml`` :
```
Comment (steindev) [inline docs/source/usage/parameter_scans.rst]: There is no reason to add a space, no?

**Comment by steindev** [comment]
@JessicaTiebel Please see the comments :point_up_2:
Also, please squash before your next push. Then we can merge directly after the CI ran through.

File: docs/source/usage/parameter_scans.rst (hunk 3 above)
Comment (JessicaTiebel) [reply docs/source/usage/parameter_scans.rst]: True, this was probably by accident. This bullet point is rendered in bold letters in readthedocs and I thought I could somehow change this, but apparently this is some sort of convention for bullet points with sub-bullet points.

**Comment by BrianMarre** [comment]
line 78 in `parameter_scan.rst`, typo: `simualtion` should be changed to `simulation`

**Comment by BrianMarre** [comment]
And I t would be nice to show an example csv file in the documentation

Otherwise good documentation, good work!

**Comment by JessicaTiebel** [comment]
I implemented @BrianMarre suggestions and will check if the read the docs renders correctly and if so will squash again.

**Review by steindev — APPROVED** [review]

**Pull request merged**